week1/armstrong.rb

```diff
@@ -9,5 +9,19 @@
 # and `false` otherwise.
 
 def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
+  len=n.to_s.length
+  n=n.to_i
+  num=n
+  sum=0
+  while n!=0
```

Comment:
This goes into an infinite loop when `n` < 0. Add a line of code to ensure that `n` is always positive. If `n` is negative, then the number isn't an Armstrong number.